skaffold.yaml

```diff
@@ -81,64 +81,75 @@ deploy:
 
 profiles:
   - name: dev
-    deploy:
-      helm:
-        releases:
-          - name: im-group-whoami-{{ .ENVIRONMENT }}
-            namespace: whoami
-            createNamespace: true
-            remoteChart: im-group
-            repo: https://dhis2-sre.github.io/im-group
-            version: 0.3.0
-            setValueTemplates:
-              serviceAccount:
-                name: im-manager-{{ .ENVIRONMENT }}
-                namespace: instance-manager-{{ .ENVIRONMENT }}
+    patches:
+      - op: add
+        path: /deploy/helm/releases/-
+        value:
+          name: im-group-whoami-{{ .ENVIRONMENT }}
```

Comment:
Since we're only changing name and namespace here, we've a lot of duplicated code. Could we use a yaml anchor to simplify things?